.travis.yml

```diff
@@ -11,7 +11,7 @@ before_install:
 script:
 - docker build --target test --tag todo-app:test .
 - docker run --mount type=bind,source="$(pwd)"/todo_app,target=/app/todo_app todo-app:test  todo_app/tests/
-- docker run --env MONGO_USERNAME=$MONGO_USERNAME --env MONGO_PASSWORD=$MONGO_PASSWORD --env MONGO_DB=$MONGO_DB --env MONGO_URL=$MONGO_URL --mount type=bind,source="$(pwd)"/todo_app,target=/app/todo_app todo-app:test  todo_app/tests_e2e/
+- docker run --env GITHUB_REDIRECT_URI=$GITHUB_REDIRECT_URI --env GITHUB_CLIENT_ID=$GITHUB_CLIENT_ID --env GITHUB_CLIENT_SECRET=$GITHUB_CLIENT_SECRET --env ROLEWRITER_USER=$ROLEWRITER_USER --env MONGO_USERNAME=$MONGO_USERNAME --env MONGO_PASSWORD=$MONGO_PASSWORD --env MONGO_DB=$MONGO_DB --env MONGO_URL=$MONGO_URL --mount type=bind,source="$(pwd)"/todo_app,target=/app/todo_app todo-app:test  todo_app/tests_e2e/
```

Comment:
Not important, but you can make this a bit more concise when passing through environment variables with the same name in both environments, e.g.
`docker run --env GITHUB_REDIRECT_URL --env GITHUB_CLIENT_ID ... ` will automatically pick up the local variables of the same name